draft-ietf-quic-load-balancers.md

```diff
@@ -1297,13 +1297,21 @@ that share the same cryptographic context for Stateless Reset tokens. As QUIC-LB
 requires deterministic routing of DCIDs over the life of a connection, it is a
 sufficient means of avoiding an Oracle without additional measures.
 
+Note also that when a server starts using a new QUIC-LB config rotation
+codepoint, new CIDs might not be unique with respect to previous configurations
+that occupied that codepoint, and therefore different clients may have observed
+the same CID and stateless reset token. A straightforward method of managing
+stateless reset keys is to maintain a separate key for each config rotation
+codepoint, and replace each key when the configuration for that codepoint
+changes. Thus, a server transitions from one config to another, it will be able
+to generate correct tokens for connections using either type of CID.
+
 ## Connection ID Entropy
 
 The Stream Cipher and Block Cipher algorithms need to generate different cipher
 text for each generated Connection ID instance to protect the Server ID. To
-do so, at least four octets of the Block Cipher CID and at least eight octets
-of the Stream Cipher CID are reserved for a nonce that, if used only once, will
-result in unique cipher text for each Connection ID.
+do so, at least four octets of the CID are reserved for a nonce that, if used
```

Comment:
This makes it look like you're reducing the minimum nonce size for both stream and block ciphers.  Is that correct?

Reply:
No, we're converging the minimum to 4 for all cases (it was already at 4 for BCCID)